From: Chad Monroe <chad@monroe.io>
Date: Mon, 24 Nov 2025 14:32:34 -0800
Subject: [PATCH] wifi-scripts: disable FT-PSK in WPA2/3 mixed mode

FT-PSK causes compatibility issues with WPA2-only clients in mixed mode.
WPA2 clients will still connect but cannot use fast roaming.

Fixes: SDG-3228

Signed-off-by: Chad Monroe <chad@monroe.io>
---
 src/lib/netifd/hostapd.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/lib/netifd/hostapd.sh
+++ b/src/lib/netifd/hostapd.sh
@@ -71,7 +71,7 @@ hostapd_append_wpa_key_mgmt() {
 			[ "$rsn_override" -gt 1 ] && wpa_key_mgmt=
 			[ "$band" = "6g" ] || {
 				append wpa_key_mgmt "WPA-PSK"
-				[ "${ieee80211r:-0}" -gt 0 ] && append wpa_key_mgmt "FT-PSK"
+				#[ "${ieee80211r:-0}" -gt 0 ] && append wpa_key_mgmt "FT-PSK"
 				[ "${ieee80211w:-0}" -gt 0 ] && append wpa_key_mgmt "WPA-PSK-SHA256"
 			}
 		;;
